cluster/images/kubelet-checker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox
```

Comment:
Should we use alpine instead of busybox (as @kelseyhightower seems to recommend for shell scripts)?